deploy/kubernetes-1.19-alpha/direct/pmem-csi.yaml

```diff
@@ -542,7 +542,7 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
-        image: k8s.gcr.io/sig-storage/csi-provisioner:v2.1.1
+        image: k8s.gcr.io/sig-storage/csi-provisioner:v2.2.1
```

Comment:
csi-provisioner:v2.2.1 no longer supports the storage capacity alpha API, i.e. this update here cannot work.

I'll close this PR and do the sidecar update together with introducing a deployment for 1.21 where storage capacity tracking then is enabled by default.